src/BehaviorsSDKNative/Microsoft.Xaml.Interactions.Design/MetadataTableProvider.DesignerIsolation.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+
+using System;
+using Microsoft.Xaml.Interactions.Core;
+using Microsoft.Xaml.Interactions.Media;
+
+namespace Microsoft.Xaml.Interactivity.Design
+{
+    partial class MetadataTableProvider
+    {
+        private void AddAttribute(Type type, Attribute attribute)
+        {
+            attributeTableBuilder.AddCustomAttributes(type, attribute);
+        }
+
+        private void AddAttributes(Type type, params Attribute[] attributes)
+        {
+            foreach (Attribute attribute in attributes)
+            {
+                AddAttribute(type, attribute);
+            }
+        }
+
+        private void AddAttribute(Type type, string propertyName, Attribute attribute)
+        {
+            attributeTableBuilder.AddCustomAttributes(type, propertyName, attribute);
+        }
+
+        private void AddAttributes(Type type, string propertyName, params Attribute[] attributes)
+        {
+            foreach (Attribute attribute in attributes)
+            {
+                AddAttribute(type, propertyName, attribute);
+            }
+        }
+
+        /// <summary>
+        /// This class contains the types used by the older Extensibility APIs.
+        /// </summary>
+        private static class Targets
+        {
+            internal static readonly Type IncrementalUpdateBehavior = typeof(IncrementalUpdateBehavior);
+            internal static readonly Type EventTriggerBehavior = typeof(EventTriggerBehavior);
+            //internal static readonly Type DataTriggerBehavior = typeof(DataTriggerBehavior);
+            //internal static readonly Type ChangePropertyAction = typeof(ChangePropertyAction);
+            internal static readonly Type InvokeCommandAction = typeof(InvokeCommandAction);
```

Comment:
Why are some of these commented out?

Reply:
We don't have native implementations of those so I believe it would be helpful to leave those lines there for later.